test/e2e/common.go

```diff
@@ -81,6 +82,20 @@ func logTable(title string, rows [][]string) {
 	w.Flush()
 }
 
+// getSha256Hash return sha256 hash of given file.
+func getSha256Hash(filename string) ([]byte, error) {
+	file, err := os.Open(filepath.Clean(filename))
```

Comment:
File is not closed anywhere. Maybe add `defer file.Close()`

Reply:
done